src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const LENGTH = 'length';
```

Comment:
Currently used names for these constants don't explain their meanings good enough.
In such cases I'd advise to define them as properties of an object, like this:
```
const SORTING_OPTIONS = {
  BY_LENGTH: 'length',
  ALPHABETICALLY: 'alphabet'
};
```
P.S. You'll have chance t get used to this approach when you learn TypeScript.